gui/VolumeOptions.cxx

```diff
@@ -241,6 +241,14 @@ void VolumeOptions::setCurrentFrame(int frame)
   d->colorizeSurfaceOptions->setCurrentFrame(frame);
 }
 
+//-----------------------------------------------------------------------------
+void VolumeOptions::setColorSurface(bool enabled)
```

Comment:
Not super-important, but you might consider:
```suggestion
void VolumeOptions::setSurfaceColored(bool enabled)
```
...or "setSurfaceColor{ing,ized,ization}".